packages/auction-widget/src/utils.tsx

```diff
@@ -51,6 +51,14 @@ function formatDate(date: number): string {
   return parseDate(date).toLocaleString();
 }
 
+/**
+ * Parse a string to check if it is a valid URL
+ */
+function is_valid_url(url: string): boolean {
+    try { return Boolean(new URL(url)); }
+    catch(e){ return false; }
+}
```

Comment:
```suggestion
function isValidUrl(url: string): boolean {
    try {
        new URL(url);
        return true; 
    } catch {
        return false;
    }
}
```
I think camel case might be more appropriate.
Also this looks a bit more readable to me.

Reply:
Oops, a bad habit from programming in Rust and Elixir!